messages/publish.json

```diff
@@ -0,0 +1,18 @@
+{
+  "description": "publishes an Experience Builder site to make it live\nWhen you publish an Experience Builder site for the first time, you make the site’s URL live and enable login access for site members.\n\nAdditionally, to send a welcome email to all site members, you must activate the site. (Activation is also required to successfully set up SEO for Experience Builder sites.) To activate a site, update the status field of the Network type in the Metadata API. Alternatively, in Experience Workspaces, go to Administration | Settings, and click Activate.\n\nSubsequently, each time you publish the site, you update the live site with all changes made to the site since it was last published.\n\nAn email notification informs you when your changes are live.",
+  "longDescription": "Publishes your Experience Builder site to make it live. Each time you publish it, you update the live site with the most recent updates.",
```

Comment:
I *think* long command descriptions (as opposed to long _flag_ descriptions) are now ignored.  So we pack all the command description into "description".  Ack, not sure if that makes sense.  In any case, I'm pretty sure you can just remove this property because it's not showing up in either the --help command output or the generatred online CLI reference.  Ping me if this makes no sense.  It's a weird legacy thing.